.gitignore

```diff
@@ -1,4 +1,10 @@
 /.vs
 /bin
 /lib
-/obj
\ No newline at end of file
+/obj
+/packages/System.Composition.AttributedModel.5.0.1/lib
+/Resources/Microsoft.Dynamics.Nav.CodeAnalysis.Workspaces.dll
+/Resources/Microsoft.Dynamics.Nav.CodeAnalysis.dll
+/Resources/Microsoft.Dynamics.Nav.Analyzers.Common.dll
+/Resources/Microsoft.CodeAnalysis.dll
+/packages/System.Composition.AttributedModel.5.0.1
```

Comment:
Resources and packages should be completely ignored I think

Reply:
Yes, isn't needed after thinking about it :)